packages/techdocs-cli/CHANGELOG.md

```diff
@@ -0,0 +1,70 @@
+# @techdocs/cli
+
```

Comment:
Yeah, good catch.  My understanding is that the release workflow will just deploy whatever package version is in here the first time (because from the monorepo's perspective, it's a net-new package).

So I believe we should actually bump the version to `0.8.5` in `package.json`, fill in the missing changesets, and move the existing changeset out from the `.changeset` folder and into the log here.